api/Window.json

```diff
@@ -1694,54 +1694,6 @@
           }
         }
       },
-      "dialogArguments": {
```

Comment:
I'd be happy to approve this removal if put in a separate PR.

Reply:
#12704